firmware/common2015/utils/logger/logger.cpp

```diff
@@ -17,6 +17,10 @@ uint8_t rjLogLevel;
 
 Mutex log_mutex;
 
+LocalFileSystem local2("local");
+FILE* fp = NULL;
```

Comment:
Since `fp` is only used inside the `log()` function, it'd be better to declare it in there rather than globally.